src/lib/y2network/autoinst_profile/route_section.rb

```diff
@@ -70,6 +70,20 @@ def self.new_from_network(route)
         result
       end
 
+      # Method used by {.new_from_hashes} to populate the attributes.
+      #
+      # @parm hash [Hash] see {.new_from_hashes}
+      # @return [Boolean]
+      def init_from_hashes(hash)
+        @destination = destination_from_hash(hash)
+        @gateway = gateway_from_hash(hash)
+        @netmask = netmask_from_hash(hash)
+        @device = device_from_hash(hash)
+        @extrapara = hash["extrapara"]
```

Comment:
why not even extrapara_from_hash when everything else is done that way?

Reply:
Basically because extrapara is ommited while the rest of the params declare it of "-" when no present. So extrapara does not need a special handling although we could add it if needed.